scripts/photon-settings.sh

```diff
@@ -13,6 +13,7 @@ echo "net.ipv6.conf.all.disable_ipv6 = 1" >> /etc/sysctl.conf
 
 echo '> Applying latest Updates...'
 tdnf -y update
+tdnf upgrade linux-esx
```

Comment:
Any packages pulled in which are changing/affecting our packages used for VEBA? (Just curious about potential regression holes)

Reply:
This was recommended per Antrea folks or else we ran into an issue. Given basic functionality has been tested in VEBA, I don't believe this has caused any regression